### PR TITLE
 Minor grammar fixes in `contributing.rst`

### PR DESCRIPTION
**Pull request by krakxn**
## Category

Documentation 

## Description

Minor grammar fixes in `contributing.rst`.

*no doggo T_T*

### CODE REVIEW

**Review by charles-cooper — CHANGES_REQUESTED** [review]
@krakxn would it be possible to bundle up all these changes into a single PR? it will make it easier on reviewers so they can review all at once.

**Comment by krakxn** [comment]
@charles-cooper Sorry, but is it okay if I follow your suggestion after these? As I did not make a lot of changes, it would be quick to review.

I will bundle it up the next time!

File: docs/contributing.rst [hunk 1]
```diff
@@ -69,9 +69,9 @@ and instead, ``git rebase`` your branch.
 Implementing New Features
 -------------------------
 
-If you are writing a new feature, please ensure you write appropriate Pytest test cases and place them under ``tests/``.
+If you are writing a new feature, please ensure you write appropriate Pytest test-cases and place them under ``tests/``.
```
Comment (charles-cooper) [inline docs/contributing.rst:72]: not sure test cases needs to be hyphenated

File: docs/contributing.rst [hunk 2]
```diff
@@ -69,9 +69,9 @@ and instead, ``git rebase`` your branch.
...
 
-If you are making a larger change, please consult first with the `Ethereum Python Dev Discord <https://discord.gg/abJEARdx3Q>`_ ``#vyper`` channel.
+If you are making a larger change, please consult first with the `Ethereum Python Dev Discord <https://discord.gg/abJEARdx3Q>`_ via ``#vyper`` channel.
```
Comment (charles-cooper) [inline docs/contributing.rst:74]: we should probably update this to the dedicated discord - https://discord.gg/6tw7PTM7C2 / #compiler-dev channel

Comment (krakxn) [reply docs/contributing.rst:74]: Okay, I will update it!

File: docs/contributing.rst (hunk 1 above)
Comment (krakxn) [reply docs/contributing.rst:72]: You are right, I will change it.

File: docs/contributing.rst [hunk 3]
```diff
@@ -42,8 +42,8 @@ reporting issues, please mention the following details:
 * What was the result of the issue
 * What the expected behaviour is
 
-Reducing the source code that caused the issue to a bare minimum is always
-very helpful and sometimes even clarifies a misunderstanding.
+Reducing the source code that caused the issue to a bare-minimum is always
```
Comment (charles-cooper) [inline docs/contributing.rst:45]: I don't think "bare minimum" should be hyphenated

**Comment by charles-cooper** [comment]
closing as stale on account of unresolved merge conflicts and unresolved review feedback

**Pull request closed without merging**